scripts/config-additional-node.sh

```diff
@@ -30,6 +32,21 @@ TIMESTAMP=`$CURL -X POST -d "" \
    |& tee -a $LOG \
    | grep "last-startup" \
    | sed 's%^.*<last-startup.*>\(.*\)</last-startup>.*$%\1%'`
+for i in `seq 1 ${N_RETRY}`; do
```

Comment:
Functionally it works fine. However curl command appears twice here which could be write to just have one appearance.
```
for i in `seq 1 ${N_RETRY}`; do
  TIMESTAMP=`$CURL -X POST \
    -H "Content-type: application/x-www-form-urlencoded" \
    --data "admin-username=${USER}" --data-urlencode "admin-password=${PASS}" \
    --data "realm=${SEC_REALM}" \
    http://${BOOTSTRAP_HOST}:8001/admin/v1/instance-admin \
    | tee -a $LOG \
    | grep "last-startup" \
    | sed 's%^.*<last-startup.*>\(.*\)</last-startup>.*$%\1%'`
  if [ "$TIMESTAMP" == "" ]; then
    WARN "Timestart wasnt retrieved. Retry in $RETRY_INTERVAL seconds"
    sleep $RETRY_INTERVAL
  else
    INFO "Timestamp retrieved"
    break
  fi
done
```